packages/components/src/Status/Status.component.js

```diff
@@ -59,28 +59,25 @@ function renderIcon(status, icon, progress) {
 		}
 		return <CircularProgress size={'small'} percent={progress} />;
 	}
-	return (icon && <Icon name={icon} />);
+	return icon && <Icon name={icon} />;
 }
 
-function Status(props) {
-	const {
-		status,
-		label,
-		icon,
-		actions,
-		progress,
-	} = props;
+export function Status(props) {
+	const { status, label, icon, actions, progress } = props;
```

Comment:
Please put that spread inside the function arguments declaration. `props` isn't reused anywhere in the function.

Reply:
done